### PR TITLE
flesh out changelog with changes in sub-packages

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

**Comment by Stebalien** [comment]
There isn't much to review here, I'm just looking for a "this looks sane"

File: CHANGELOG.md [hunk 1]
```diff
@@ -384,6 +381,123 @@ Cleanup:
...
+  - add standard ci scripts ([ipfs/go-ipfs-cmdkit#23](https://github.com/ipfs/go-ipfs-cmdkit/pull/23))
+  - only count size for regular files ([ipfs/go-ipfs-cmdkit#25](https://github.com/ipfs/go-ipfs-cmdkit/pull/25))
+  - Create Jenkinsfile ([ipfs/go-ipfs-cmdkit#16](https://github.com/ipfs/go-ipfs-cmdkit/pull/16))
+  - Feat: add WebFile File implementation. ([ipfs/go-ipfs-cmdkit#26](https://github.com/ipfs/go-ipfs-cmdkit/pull/26))
+  - feat(type): fix issue #28 ([ipfs/go-ipfs-cmdkit#29](https://github.com/ipfs/go-ipfs-cmdkit/pull/29))
+  - Extract files package ([ipfs/go-ipfs-cmdkit#31](https://github.com/ipfs/go-ipfs-cmdkit/pull/31))
+
+github.com/ipfs/go-ds-badger
+  - update protobuf ([ipfs/go-ds-badger#26](https://github.com/ipfs/go-ds-badger/pull/26))
+  - exported type datastore => Datastore ([ipfs/go-ds-badger#1](https://github.com/ipfs/go-ds-badger/pull/1))
```
Comment (schomatis) [inline CHANGELOG.md]: These two look a bit old.

**Comment by schomatis** [comment]
We could add a couple of the PRs of `go-unixfs`, is there a script you use to generate the list? If not I'll do it manually since there are only a few ones worth noting.

**Comment by Stebalien** [comment]
I hacked together an "ephemeral" script (not good enough to keep). It missed those because those packages simply didn't exist in the previous release. Thanks for noticing that, I'll pull those changes.

**Comment by Stebalien** [comment]
> I hacked together 

Basically, I used `jq` to diff the old `package.json` and the new one, and then modified `bin/mkreleasenotes` to pull changes from arbitrary repos.

File: CHANGELOG.md (hunk 1 above)
Comment (schomatis) [reply CHANGELOG.md]: Yes, I'm somewhat confused, those two (`pull/1` and `pull/2`) seem to be merged in 2017 but at the end of the PRs says that they were referenced from another PR of July.

**Pull request merged**